examples/migrations/langgraph-checkpoints/artifacts/legacy/memanto-roundtrip-okf/sessions/langgraph-migration-demo_2026-07-22_unknown_summary.md

```
@@ -0,0 +1,172 @@
+# Session Summary for langgraph-migration-demo
+**Session ID:** `unknown`
+
+---
+
+### [2026-07-22 19:09:51] [ARTIFACT] LangGraph transcript: product-launch
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)